CanonicalDescription/test2/README.md

```diff
@@ -0,0 +1,46 @@
+Tests 2 : Two reactors over 120 years
+=============================================
+
+120 year simulation, 2 reactors, fixed fuel recipe, 120 years operation to be defined: reactor power, fuel burnup and associated recipes, (re)fueling strategy
+
+Definition
+----------------
+120 year simulation, 2 reactors, fixed fuel recipe, 120 years operation
+
+Reactor : 
+* Type: LWR, 
+* **Fuel**: UOX 3.25%w enriched uranium, 
+* **Mass**: 82 tons (Oxyde Metal),
+* **Thermal Power**: 2875e6 W -- **Efficiency**: 34%,  
+* **Discharge burn-up** : 43.03 GWd/t -- Full Cycle length 3y, 
+* **Capacity factor**: 1, 
+* **Suggested recipes**:
+  * **Fresh UOX** :
+    * <sup>235</sup>U   :    3.25%w
+    * <sup>238</sup>U   :   96.75%w
+    * <sup>16</sup>O    :   13.44%w \
+    * the %w are normalize to the quantity of uranium... but because the reactor fuel mass is given in oxyde metal, one provides also the corresponding oxygen quantity
+
+  * **Used UOX** @43.03 GWd/t (atomic):
```

Comment:
the standard abbreviation for tonne is `t`.
but I agree it could be confusing... I can change it for Mg or tonne... I don't know wich one is better...